src/script/auth/page/OauthPermissions.styles.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CSSObject} from '@emotion/react';
+
+import {COLOR_V2} from '@wireapp/react-ui-kit';
+
+export const getTeamImageCSS = (): CSSObject => ({
```

Comment:
is there any particular motivation in using functions instead of plain objects here. 
There doesn't seem to be any dynamic values in none of those. Just wondering if you see a particular benefit in having functions?

Reply:
ah no sorry i just looked at another file and copied the syntax since i haven't done this in a while. lets make them objects